app/src/test/java/com/sdp/movemeet/Backend/Firebase/FirebaseDB/FirebaseDBActivityManagerTest.java

```diff
@@ -0,0 +1,334 @@
+package com.sdp.movemeet.Backend.Firebase.FirebaseDB;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.api.Backend;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.sdp.movemeet.Activity.Activity;
+import com.sdp.movemeet.Activity.ActivityTest;
+import com.sdp.movemeet.Backend.Serialization.ActivitySerializer;
+import com.sdp.movemeet.Backend.Serialization.ActivitySerializerTest;
+import com.sdp.movemeet.Backend.Serialization.BackendSerializer;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.Executor;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class FirebaseDBActivityManagerTest {
+
+    FirebaseDatabase db;
+    FirebaseDBManager<Activity> fam;
+    BackendSerializer<Activity> serializer;
+
+    DatabaseReference dbRef;
+    DataSnapshot getSnap;
+
+    Activity activity;
+    String path;
+
+    Task<Void> addTask = new Task<Void>() {
+        @Override
+        public boolean isComplete() {
+            return true;
+        }
+
+        @Override
+        public boolean isSuccessful() {
+            return true;
+        }
+
+        @Override
+        public boolean isCanceled() {
+            return false;
+        }
+
+        @Override
+        public Void getResult() {
+            return null;
```

Comment:
Are all those methods really supposed to return null, or is it just not implemented yet ?